src/microcosm.js

```diff
@@ -260,6 +260,22 @@ class Microcosm extends Emitter {
     // Remove all listeners
     this.removeAllListeners()
   }
+
+  parallel(actions) {
+    return this.append('GROUP').link(actions)
+  }
+
+  sequence(prepared) {
+    let payload
+
+    let sequence = () => function * () {
+      for (var i = 0; i < prepared.length; i++) {
+        payload = yield prepared[i](payload)
+      }
+    }
+
+    return this.push(sequence)
+  }
```

Comment:
I really like how clean this is, but if we use generators inside of Microcosm, we'll need to pull in the [regenerator runtime](https://github.com/facebook/regenerator) that makes them possible in older browsers.

It's like 600 bytes, but I'd rather let people patch their own run time in. How could we write this so that it does not depend on generators? Could we call `reduce` on prepared?